tests/Test/Algolia/DummyManualIndexer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tests\AppBundle\Test\Algolia;
+
+use AppBundle\Algolia\ManualIndexerInterface;
+use Psr\Log\LoggerInterface;
+
+class DummyManualIndexer implements ManualIndexerInterface
+{
+    private $logger;
+
+    public function __construct(LoggerInterface $logger = null)
+    {
+        $this->logger = $logger;
+    }
+
+    public function index($entities, array $options = []): void
+    {
+        if ($this->logger) {
+            if (is_array($entities)) {
+                $this->logger->info(sprintf('[algolia] indexing %d entities.', count($entities)));
```

Comment:
Perhaps the logger context (2nd argument) is smart enough to dump `$entities` in a generic way so that this code could be simplified a lot.

Reply:
i dont want to dump an array of entities if it's the case. i only want to know the count.
and i dont want to dump the entity if it's the case... only want to know which type of entity.